### PR TITLE
Extend START, END, READY controls to allow BOOL type

### PR DESCRIPTION
**Pull request by krishung5**

### CODE REVIEW

File: include/triton/backend/backend_common.h [hunk 1]
```diff
@@ -351,16 +351,20 @@ TRITONSERVER_Error* CheckAllowedModelOutput(
 /// \param fp32_true_value Returns the float value for true if
 /// the tensor type is FP32.
 /// \param int32_false_value Returns the int value for false if
-/// the tensor type is FP32.
+/// the tensor type is int32.
```
Comment (CoderHam) [inline include/triton/backend/backend_common.h:354]: this should be INT32 to be consistent

File: include/triton/backend/backend_common.h [hunk 2]
```diff
@@ -351,16 +351,20 @@ TRITONSERVER_Error* CheckAllowedModelOutput(
...
 /// \param int32_true_value Returns the int value for true if
-/// the tensor type is FP32.
+/// the tensor type is int32.
```
Comment (CoderHam) [inline include/triton/backend/backend_common.h:356]: Here too

File: src/backend_common.cc [hunk 3]
```diff
@@ -504,20 +535,24 @@ GetTypedSequenceControlProperties(
...
+                (c.Find("bool_false_true", &bool_false_true) &&
+                 (bool_false_true.ArraySize() > 0));
+            if (found_fp32 || found_int32 || found_bool) {
               return TRITONSERVER_ErrorNew(
                   TRITONSERVER_ERROR_INVALID_ARG,
                   (std::string(
-                       "sequence batching must not specify either "
-                       "'int32_false_true' "
-                       "nor 'fp32_false_true' for " +
+                       "sequence batching must not specify neither "
```
Comment (CoderHam) [inline src/backend_common.cc:553]: not specify either ... or ...

**Review by CoderHam — CHANGES_REQUESTED** [review]

File: src/backend_common.cc (hunk 3 above)
Comment (krishung5) [reply src/backend_common.cc]: Updated

**Review by CoderHam — APPROVED** [review]

**Review by deadeyegoodwin — DISMISSED** [review]

**Review by deadeyegoodwin — APPROVED** [review]

**Pull request merged**